app/src/main/java/com/lokico/PSWind/NOAA.java

```diff
@@ -1,17 +1,108 @@
 package com.lokico.PSWind;
 
+import android.graphics.Color;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
 
 public class NOAA extends AppCompatActivity {
 
+    private static final String TAG = "NOAA";
+    RequestQueue mRequestQueue;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_noa);
+
+        mRequestQueue = Volley.newRequestQueue(getApplicationContext());
```

Comment:
I agree on adding enough comments so the future you can understand what the old you was thinking.
I have known folks who dislike all comments though and just rely on good variable/method naming.

I'll add an item to my check list to make sure I have commented enough.